packages/python/build.mjs

```diff
@@ -0,0 +1,3 @@
+import { esbuild } from '../../utils/build.mjs';
+
+await esbuild({ bundle: true, external: ['@vercel/build-utils'] });
```

Comment:
Don't you need to mark all the `dependencies` in package.json as external too?

Reply:
Non-issue... using the new builder-build.mjs script